lib/galaxy/job_metrics/instrumenters/cgroup.py

```diff
@@ -27,14 +48,35 @@
     "memory.oom_control.oom_kill_disable": "OOM Control enabled",
     "memory.oom_control.under_oom": "Was OOM Killer active?",
     "cpuacct.usage": "CPU Time",
+    # cgroupsv2
+    "memory.events.low": "Number of times the cgroup was reclaimed due to high memory pressure even though its usage is under the low "
```

Comment:
This all looks like a nice thing to stick into a model and do on the client-side ;).

Reply:
Sure, someone is welcome to overhaul it in that manner, but it's unlikely to be me. :smile: